src/BundlerMinifier.TagHelpers/BundleExtensions.cs

```diff
@@ -1,6 +1,9 @@
 using System;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+#if !USEWEBHOSTENV
```

Comment:
`if !NETSTANDARD2_0`? Then you don't need a custom define.

Reply:
I tried, it does not work. This is strange though, `NETSTANDARD2_0` is not defined in netcoreapp3.0 when looking with visual studio. But if I build with `dotnet build`, it get defined even on netcoreapp3.0